doc/tutorials/compute_shader/main.py

```diff
@@ -16,6 +17,8 @@
 GRAPH_HEIGHT = 120
 GRAPH_MARGIN = 5
 
+CURRENT_DIR = Path(__file__).parent.resolve()
+
```

Comment:
The tests will not run if the full path is not used, thats why I added the use of `pathlib.Path`.

Reply:
Add a comment that either says don't worry about it or gets the files location.

Reply:
Change the `cwd` before running the example in unit tests? That's the least intusive.

I would order solutions like this (best to worst)
* Change `cwd` in unit test before importing/running the module (No code changes needed)
* Use `pathlib` (The code change with the least amount of consequences)
* Change `cwd` in the example (This is not any less "magical" than the pathlib solution, but it can create other problems for the user)

A 4th solution was implementing a resouce handle `:this:` (or `:here:`) that would use introspection to figure out what module the user attepmted to load the resource from.